vendor.mod

```diff
@@ -93,7 +93,7 @@ require (
 	github.com/vbatts/tar-split v0.11.5
 	github.com/vishvananda/netlink v1.2.1-beta.2
 	github.com/vishvananda/netns v0.0.4
-	go.etcd.io/bbolt v1.3.9
+	go.etcd.io/bbolt v1.3.10
```

Comment:
Wait; why does this show v1.3.9 before this PR was merged? We merged these PR's earlier this week; did something revert those?

- https://github.com/moby/moby/pull/47933
- https://github.com/moby/moby/pull/47934

Reply:
Perhaps GitHub shows the wrong base? Git blame (through GitHub) shows those PRs, not this one
<img width="856" alt="Screenshot 2024-06-11 at 23 52 02" src="https://github.com/moby/moby/assets/1804568/346c05f2-1393-4c8f-bfb9-71f24afc1a82">
<img width="783" alt="Screenshot 2024-06-11 at 23 52 15" src="https://github.com/moby/moby/assets/1804568/8271d714-8431-418f-b087-c34e1b9af08f">

Reply:
Yeah, I forgot to fast-forward to updated master before I did update. The commit was on top of previous vendor 5 days ago and that's what github shows. The merge commit shows the correct diff https://github.com/moby/moby/commit/c6b12f72a0bbe0d0f199e616bdbdfae21d2f826b

Reply:
Ah, yes; usually such cases would show the other commits as well, so that's what probably confused me.

No harm done! I just was confused for a bit ☺️🥹